libs/books/feature/src/lib/book-search/book-search.component.ts

```diff
@@ -60,4 +71,20 @@ export class BookSearchComponent implements OnInit {
       this.store.dispatch(clearSearch());
     }
   }
+
+  unDoAddingToList(bo: Book, action) {
+    this.snackBarRef = this.snackBar.open('book added reading list', action, {
```

Comment:
Move all the strings to constants

`book added reading list` and duration value